integration_test/third_party_apps_data/applications/nginx/exercise

```diff
@@ -0,0 +1,9 @@
+set -e
+
+# Create a file that nginx can't read.
+touch /var/www/html/forbidden.html
+chmod o-r /var/www/html/forbidden.html
+
+# Then request that file through nginx. This is meant to generate an entry in
+# the access log and the error log.
+! curl http://localhost:8080/forbidden.html
```

Comment:
What is the `!` for here? `curl` will exit status 0 regardless of what the status code is.

Reply:
Oh, it's because I was curling the wrong port haha. I see what  you mean now.

Reply:
![image](https://user-images.githubusercontent.com/115761/131558436-83cf45dd-d042-459d-bcad-b4feebfa2a50.png)